### ISSUE TITLE
Right nav formatting problem when page has no TOC

### ISSUE THREAD

**Issue by chalin** [open-telemetry/opentelemetry.io#1102]
E.g., see:

- https://opentelemetry.io/docs/getting-started/
- https://opentelemetry.io/blog/2023/integrations/
- https://opentelemetry.io/docs/instrumentation/ruby/events/

Here's an example:

> <img width="332" alt="Screen Shot 2022-02-04 at 9 37 27 AM" src="https://user-images.githubusercontent.com/4140793/152547496-c734ab7e-4700-4a96-87ac-f1c73b5ade96.png">

Notice the space missing between the page-meta links and the top-nav.

Might this be a recently introduced docsy issue?

Here is the corresponding Docsy issue:

- https://github.com/google/docsy/issues/1573

**Comment by svrnm** [comment]
I don't think this is still an issue? @chalin

**Comment by chalin** [comment]
Yes it is AFAIK, and I think I came across the source of the problem while working in the Docsy repo. I'll post an update here as soon as I can.

**Comment by svrnm** [comment]
TY!

**Comment by chalin** [comment]
Notes:

- While a workaround was proposed in #3508 ...
- It might be unnecessary following the latest Docsy upgrade that is fixing #3157. Quickly, I've just noticed that for a few pages that had the right-nav problem, that problem does away once the left-nav height is fixed.

To be investigated further!

**Comment by chalin** [comment]
Closed by:

- ~#3519~
- #9135
